Tasks/DecryptFile/decrypt.ps1

```diff
@@ -0,0 +1,60 @@
+#
+#  Copyright (c) Microsoft. All rights reserved.  
+#  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+param(
+    [string]$cipher,
+    [string]$inFile,
+    [string]$passphrase,
+    [string]$outFile,
+    [string]$cwd
+)
+
+Write-Verbose "Entering script DecryptFile.ps1"
+Write-Host "cipher = $cipher"
```

Comment:
Sorry, I think I caused some confusion. Typically the pattern we follow is to not localize write-verbose. The thinking is that we consider this diagnostic information and if a customer has to send in their diagnostic logs, then the folks analyzing the diagnostic logs prefer to see English diagnostic info :)